src/pocketmine/Player.php

```diff
@@ -3960,4 +3965,8 @@ public function getLoaderId() : int{
 	public function isLoaderActive() : bool{
 		return $this->isConnected();
 	}
+
+	public function getDeviceOS() : int{
+	    return $this->deviceOS;
```

Comment:
formatting

Reply:
The formatting is fine...

Reply:
no, it's not. this line and the line below have incorrect indentation. It should use tabs, not spaces.